phys2bids/physio_obj.py

```diff
@@ -456,7 +456,14 @@ def check_trigger_amount(self, thr=None, num_timepoints_expected=0, tr=0):
         # Use the trigger channel to find the TRs,
         # comparing it to a given threshold.
         trigger = self.timeseries[self.trigger_idx]
+        time = self.timeseries[0]
         LGR.info(f'The trigger is in channel {self.trigger_idx}')
+        # Check that trigger and time channels have the same length.
+        # If not, resample time to the length of the trigger
+        if len(time) != len(trigger):
+            time = np.linspace(time[0], time[-1], len(trigger))
```

Comment:
I believe these two lines here are not tested. Could you open an issue for this please? Or add a quick assert to the  test. Thanks!